estudios_socioeconomicos/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import render, get_object_or_404
+from django.contrib.auth.decorators import login_required, user_passes_test
+
+from captura.utils import get_study_info
+from perfiles_usuario.utils import is_capturista_or_administrador, is_capturista
+from familias.models import Integrante
+from familias.utils import total_egresos_familia, total_ingresos_familia, total_neto_familia
+from indicadores.models import Transaccion, Ingreso
+from .models import Estudio, Foto
+
+
+@login_required
+@user_passes_test(is_capturista_or_administrador)
```

Comment:
i'd suggest to use this instead:
`@user_passes_test(lambda u: is_member(u, [ADMINISTRADOR_GROUP, CAPTURISTA_GROUP]))
`
for two reasons:
1. I've done that in my pr :p https://github.com/erikiado/jp2_online/pull/82/files
2. there are no tests for is_capturista_or_administrador